torchvision/transforms/transforms.py

```diff
@@ -2104,8 +2104,12 @@ def __init__(self, alpha=50.0, sigma=5.0, interpolation=InterpolationMode.BILINE
             interpolation = _interpolation_modes_from_int(interpolation)
         self.interpolation = interpolation
 
-        if not isinstance(fill, (int, float)):
-            raise TypeError(f"fill should be int or float. Got {type(fill)}")
+        if isinstance(fill, (int, float)):
+            fill = [float(fill)]
```

Comment:
Why can't we just leave those as single scalars? Looks like the functional can handle them?

Reply:
It can't in JIT mode. See https://github.com/pytorch/vision/pull/7141#issuecomment-1406349224.